### PR TITLE
Add @90zlaya

### PR DESCRIPTION
**Pull request by 90zlaya**
Zlatan Stajic

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
 // keep it commonjs export
 module.exports = [
+  {
+    name: 'Zlatan Stajic',
+    description:
+      'M.Sc. in Computer Science. Working as a Web Developer. Creator of bit.ly/php-library.',
+    url: 'https://www.zlatanstajic.com/#uses',
```
Comment (HugoDF) [inline src/data.js:7]: This isn't a uses page

**Comment by 90zlaya** [comment]
> In order to get merged you'll need a uses page that details the tools you use (hardware/software) not just the technology services you provide

Got it, thanks.

File: src/data.js (hunk 1 above)
Comment (90zlaya) [reply src/data.js:7]: Will do on a update

Comment (90zlaya) [reply src/data.js]: @HugoDF Created /uses page

**Comment by 90zlaya** [comment]
> In order to get merged you'll need a uses page that details the tools you use (hardware/software) not just the technology services you provide

Hope all is well now. :1st_place_medal:

**Comment by HugoDF** [comment]
If you resolve the conflicts I'll merge this

**Comment by 90zlaya** [comment]
> If you resolve the conflicts I'll merge this

Resolved

**Pull request merged**